cli/command/formatter/stats.go

```diff
@@ -0,0 +1,135 @@
+package formatter
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/docker/go-units"
+)
+
+const (
+	defaultStatsTableFormat    = "table {{.Container}}\t{{.CPUPrec}}\t{{.MemUsage}}\t{{.MemPrec}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDs}}"
+	winDefaultStatsTableFormat = "table {{.Container}}\t{{.CPUPrec}}\t{{{.MemUsage}}\t{.NetIO}}\t{{.BlockIO}}"
+	emptyStatsTableFormat      = "Waiting for statistics..."
+
+	containerHeader  = "CONTAINER"
+	cpuPrecHeader    = "CPU %"
+	netIOHeader      = "NET I/O"
+	blockIOHeader    = "BLOCK I/O"
+	winMemPrecHeader = "PRIV WORKING SET"  // Used only on Window
+	memPrecHeader    = "MEM %"             // Used only on Linux
+	memUseHeader     = "MEM USAGE / LIMIT" // Used only on Linux
+	pidsHeader       = "PIDS"              // Used only on Linux
+)
+
+// ContainerStatsAttrs represents the statistics data collected from a container.
+type ContainerStatsAttrs struct {
+	Windows          bool
+	Name             string
+	CPUPercentage    float64
+	Memory           float64 // On Windows this is the private working set
+	MemoryLimit      float64 // Not used on Windows
+	MemoryPercentage float64 // Not used on Windows
+	NetworkRx        float64
+	NetworkTx        float64
+	BlockRead        float64
+	BlockWrite       float64
+	PidsCurrent      uint64 // Not used on Windows
+}
+
+// ContainerStats represents the containers statistics data.
+type ContainerStats struct {
+	Mu sync.RWMutex
```

Comment:
> @runcom I was about to suggest myself but I don't mind you fixing this.

yeah, I'll try to find some time so feel free to go ahead and take a stab at this...